eslint.config.mjs

```diff
@@ -105,15 +80,10 @@ export default [
         },
       ],
       '@typescript-eslint/no-redundant-type-constituents': 'warn',
-    },
-  },
-  {
-    files: ['apps/portals-e2e/*.ts'],
-    rules: {
-      '@typescript-eslint/no-floating-promises': 'error',
+      'barrel/avoid-barrel-files': 'warn',
```

Comment:
The only lint rule change I manually added was to add this plugin to avoid barrel files. We should remove barrel files to speed up bundling (in both dev mode and production builds) ([source](https://e18e.dev/guide/speedup.html)) -- for now, just warn against it